### DISCUSSION TITLE
LDAP Record - bypass authentication by Twill

### DISCUSSION THREAD

**Discussion by HeartlandTechie** [area17/twill#1145] [Q&A]
Ok - deploying this in an enterprise application and we use Active Directory to authenticate - I've installed Laravel/Jetstream and LDAP Record . . . all that is happy and running.   Now I'm wanting to authenticate to that for twill - any ideas or suggestions?

**Comment by ifox** [comment]
Hi @HeartlandTechie, this is doable. We have Google auth working through Laravel Socialate as an option, and we've implemented SAML based authentication on top of Twill in the past too.
